src/components/collection_voting_slab/ResearchProposalVoteSlab.js

```diff
@@ -130,17 +131,41 @@ const ResearchPurposeSummary = ({darInfo}) => {
     div();
 };
 
-export const ChairVoteInfo = ({dacVotes, isChair, isLoading}) => {
-  return div({style: styles.chairVoteInfo, isRendered: isChair && dacVotes.length > 0, datacy: 'chair-vote-info'}, [
-    h(VotesPieChart, {
-      votes: dacVotes,
-    }),
-    div(['My DAC\'s Votes (detail)']),
-    h(VoteSummaryTable, {
-      dacVotes: collapseVotesByUser(dacVotes),
-      isLoading,
-    })
-  ]);
+//this component needs algorithmResult, should it be sent as a prop or set via React.Context?
+export const ChairVoteInfo = ({dacVotes, isChair, isLoading, algorithmResult = {}}) => {
```

Comment:
Since we're passing in the votes, chair status, etc., I lean towards passing in the algorithm as another argument.